etl/steps/data/garden/wash/2024-01-06/who.py

```diff
@@ -0,0 +1,163 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+import re
+
+import numpy as np
+from owid.catalog import Table
+from owid.catalog import processing as pr
+from structlog import get_logger
+
+from etl.data_helpers import geo
+from etl.data_helpers.geo import add_regions_to_table
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+log = get_logger()
+
+REGIONS_TO_ADD = [
```

Comment:
also, maybe I am missing something but looks like you are not actually doing any regional aggregates.. maybe you used to but not anymore?

Reply:
Yeh I was going to but I didn't in the end, the guy I was talking to at the data provider sent me some regional aggregates, but forgot to remove this - thanks!